src/gui/SetupDialog.cpp

```diff
@@ -1471,6 +1509,40 @@ void SetupDialog::setBackgroundArtwork( const QString & _ba )
 
 
 
+void SetupDialog::setAutoSaveInterval( int value )
+{
+	QString minutes = value > 1 ? tr( "minutes" ) : tr( "minute" );
+
+	if( m_saveIntervalSlider->value() != value )
+	{
+		m_saveIntervalSlider->setValue( value );
```

Comment:
The downside is that the slider is spamming the function if you scroll over the limits. But the value is only updated when you hit OK anyway. Removed.